packages/devtools_app/lib/src/screens/memory/panes/diff/controller/diff_pane_controller.dart

```diff
@@ -306,7 +309,10 @@ class DerivedData extends DisposableController with AutoDisposeControllerMixin {
   /// Returns [classStats] if it matches the current filter.
   T? _filter<T extends ClassStats>(T? classStats) {
     if (classStats == null) return null;
-    if (_core.classFilter.value.apply(classStats.heapClass)) return classStats;
+    if (_core.classFilter.value.apply(
+      classStats.heapClass,
+      _core.rootPackage,
+    )) return classStats;
```

Comment:
Nit: try to avoid conditional statements with single line bodies without a scope, especially if they can't fit entirely on a single line. Using braces, even for single line bodies, reduces the likelihood of an error being introduced later during refactoring or further changes.